package.json

```diff
@@ -4,6 +4,8 @@
     "packages/*"
   ],
   "scripts": {
+    "build": "pnpm --filter vilay --filter render -r build",
+    "postinstall": "pnpm build",
```

Comment:
Do you have any particular reason for adding this?

Reply:
this helps get the repo set up more easily for first time contributors, i will have a few more tweaks like these!

you can also run `pnpm build --watch` at the root while making changes to one of these dependencies, and the example dev server reloads, it's nicer than manually building the deps each time I made a change

Reply:
if you prefer I can remove them of course, if it doesn't make sense to you